### ISSUE TITLE
Problems with right eye render

### ISSUE THREAD

**Issue by tahasabih** [NICTA/cesium-vr#14]
<img width="1249" alt="screen shot 2015-08-11 at 6 34 50 pm" src="https://cloud.githubusercontent.com/assets/6337167/9195733/bff6f390-4057-11e5-913b-2c42365ccc42.png">

Hi, there is a slight problem in the rendering in one of the eyes. When using OS X, the problem can be seen on the screen on the right eye without the rift connected, but when the rift is connected the problem occurs on the left eye. The problem occurs on the left eye on Windows as well. 

You can see the problematic render on the image attached. Any idea what might be causing this?

OS X: Oculus runtime 5.0.1
Windows: Oculus runtime V0.6.0.1

Both running Nightly with the VR add-on.

**Comment by craigsketchley** [comment]
Hey @tahasabih, thanks for this. I'll have a look into it in the next day and see if I can reproduce it.

**Comment by craigsketchley** [comment]
Hi @tahasabih, what version of Nightly are you using? Do you have a similar experience at http://mozvr.com/? I'm thinking it might be a problem with the latest version of Nightly, 43.0a1.

**Comment by tahasabih** [comment]
Yes I am using 43.0a1, but the examples over at mozvr.com are rendering correctly. I feel there is a problem with the FrustrumPatch but I have not been able to locate the issue.

**Comment by oliverheilig** [comment]
Have a similar problem with 43.0a1, Win10 + Oculus SDK 0.7. For me it's the left eye. My temporary workaround is to 

      this.context2d.drawImage(srcCanvas, **100**, 0);

**Comment by oliverheilig** [comment]
Have a similar problem, 43.0a1, Win10 + Oculus SDK 0.7. It's the right part of the left eye for me. My temporary workaround is to shift the drawImage by 100 pixels in CanvasCopy.prototype.copy 

      this.context2d.drawImage(srcCanvas, 100, 0);

**Comment by Ylannl** [comment]
Same here on 43.0a1, OS X + Oculus DK2 with SDK 0.5.0.1. Downgrading to an earlier Firefox Nightly fixes it for me (I tried 41.0).

**Comment by craigsketchley** [comment]
Cheers guys, and thanks for the workaround @oliverheilig. It seems that this is related to an issue with the current Firefox build. Here's a post from the WebVR mailing list together with a link to the filed bug: https://mail.mozilla.org/pipermail/web-vr-discuss/2015-August/000824.html
